doc/releases/migration-guide-3.7.rst

```diff
@@ -30,6 +30,10 @@ Boards
 
 * Reordered D1 and D0 in the `pro_micro` connector gpio-map for SparkFun Pro Micro RP2040 to match
   original Pro Micro definition. Out-of-tree shields must be updated to reflect this change.
+* ITE: Rename all SoC variant names, e.g., ``SOC_IT82202_AX`` is renamed to ``SOC_IT82202AX``.
```

Comment:
Should this say?
```suggestion
* ITE: Rename all SoC variant names kconfig options, e.g., ``CONFIG_SOC_IT82202_AX`` is renamed to ``CONFIG_SOC_IT82202AX``.
```
if it is only the kconfig options and nothing else that changes (i.e. only those who have custom boards or happen to use those kconfig options in their code may realize)

Reply:
Done.